src/tests/BaGet.Services.Mirror.Tests/ServiceCollectionExtensionsTest.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+using BaGet.Core.Configuration;
+using BaGet.Extensions;
+using BaGet.Services.Mirror.Extensions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BaGet.Services.Mirror.Tests
+{
+    public class ServiceCollectionExtensionsTest
+    {
+
+        [Fact]
+        public void AskServiceProviderForNotConfiguredMirrorOptions()
+        {
+            ServiceProvider provider = new ServiceCollection()
+                .AddBaGetContext() //Required
+                .AddMirrorServices() //Method Under Test!
+                .BuildServiceProvider();
+
+            InvalidOperationException expected = Assert.Throws<InvalidOperationException>(
```

Comment:
>InvalidOperationException [](start = 12, length = 25)

Please use `var` everywhere :)

Reply:
We should agree, that this point ("everywhere")  can be a long discussion between professional developers with a long list of pro and cons.

But we don't do that now ;-)  i'm following the project rules....